networking/networking.tex

```diff
@@ -806,7 +806,7 @@ \section{Layer 4: TCP Server}
 \subsection{Example Server}
 
 A working simple server example is shown below.
-Note this example is incomplete - for example the socket file descriptor remains open and memory created by \keyword{getaddrinfo} remains allocated.
+Note this example is incomplete - for example, the socket file descriptor remains open and memory created by \keyword{getaddrinfo} remains allocated.
```

Comment:
This isn't a sentence or it is an abonimation of one. Please split it into `Note this ...` and `For example,`